src/components/RequestForm.spec.js

```diff
@@ -0,0 +1,85 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import RequestForm from './RequestForm';
+
+describe('RequestForm', () => {
+  it('renders a form with three input fields, a textarea and a button to submit the form', () => {
+    render(
+      <MemoryRouter>
+        <RequestForm buttonName="POST" />
+      </MemoryRouter>
+    );
+
+    const destinationInput = screen.getByLabelText(/destination/i);
+    const descriptionTextarea = screen.getByLabelText(/description/i);
+    const submitButton = screen.getByRole('button', { name: 'POST' });
```

Comment:
You could test for the date inputs too ;)

Reply:
Thx, have added that :)